esileapclient/tests/functional/v1/test_basic.py

```diff
@@ -213,6 +213,31 @@ def test_lease_show_offer_deleted(self):
         details = esi.lease_show(self.clients['child-lessee'], lease['uuid'])
         self.assertEqual(details['status'], 'deleted')
 
+    def test_node_list_basic(self):
+        """ Tests basic functionality of "esi node list" when executed by
+             node owners
+            Test steps:
+            1) Created dummy node while Instantiating the dummy node object.
+            2) Check that node details were returned.
+            3) Check that the output of 'node list' contains the new node.
+            4) (cleanup) Delete the node created in step 1. """
```

Comment:
I think 1 and 4 aren't needed within the test, since the dummy node is managed by the test class.

Reply:
updated.